src/+otp/+pendulum/PendulumProblem.m

```diff
@@ -0,0 +1,50 @@
+classdef PendulumProblem < otp.Problem
+    % PendulumProblem A two-variable model for the trajectory of a pendulum
+    %
+    %
+    %   This system can be reduced to the following two equations
+    %
+    %   theta' = omega
+    %   omega' = -g/l * sin(theta)
+    %
+    %   where g is the gravitational constant, l is the length of
+    %   the pendulum, and theta is the angle between the pendulum and
+    %   y-axis.
+     
+    methods
+        function obj = PendulumProblem(timeSpan, y0, parameters)
+            % Constructs a problem
+            obj@otp.Problem('Pendulum', 2, timeSpan, y0, parameters);
+        end
+    end
+    
+    methods (Access = protected)
+        function onSettingsChanged(obj)
+            g = obj.Parameters.g;
+            l = obj.Parameters.l;
+            
+            obj.Rhs = otp.Rhs(@(t, y) otp.pendulum.f(t, y, g, l), ...
```

Comment:
I know that this is your first problem, and overwhelming you is not the best thing, but adding derivatives with respect to parameters would be perfect for this simple problem. df/dg and df/dl would be great trivial tests for parameter estimation.

Reply:
Might as well add Jacobian-vector products too.  Maybe even Hessian too

Reply:
and adjoint information (jac adj vp, hes asj vp) as well. For a problem as simple as this, we have to.